uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/annual-entitlements.html

```diff
@@ -0,0 +1,36 @@
+<div class="panel panel-default chr_annual-entitlements">
+  <div class="panel-heading">
+    <h3 class="panel-title">Annual Entitlements</h3>
+  </div>
+  <crm-loading show="entitlements.loaded.absencePeriods">
+    <div class="table-responsive" ng-show="entitlements.loaded.absencePeriods && entitlements.absencePeriods.length > 0">
+      <table class="table">
+        <thead>
+          <tr>
+            <th class="chr_annual-entitlements__period">Period</th>
+            <th ng-repeat="absenceType in entitlements.absenceTypes">{{absenceType.title}}</th>
+            <th class="chr_annual-entitlements__edit-entitlements"></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr ng-repeat="absencePeriod in entitlements.absencePeriods">
+            <td>{{absencePeriod.period}} <i class="fa fa-clock-o text-primary" aria-hidden="true"></i></td>
```

Comment:
It is mentioned in the ticket description that the clock item should not be added as its not supported yet, 
`Clock - not supported yet, mention in PR(From Ticket)`
So please remove this clock icon , and mention in the PR description that its not supported yet.